addons/weather/functions/fnc_calculateRoughnessLength.sqf

```diff
@@ -19,8 +19,10 @@
 #define ROUGHNESS_LENGTHS [0.0002, 0.0005, 0.0024, 0.03, 0.055, 0.1, 0.2, 0.4, 0.8, 1.6]
 
 private _windSource = _this vectorDiff ((vectorNormalized wind) vectorMultiply 25);
-
-private _nearBuildings = count (_windSource nearObjects ["Building", 50]);
+private _nearBuildings = {
+    // Filter lights - fixes high roughness on airports (#6602)
+    str _x find "light" == -1
```

Comment:
This is case sensitive.

Reply:
I know.

Reply:
Most of CUP runway lights are lowercase. With exception of `CUP_A1_NavigLight`
Vanilla Arma "Runway Edgelight" is `Land_Flush_Light_green_F` uppercase Light.
`Land_NavigLight`/`\A3\Roads_F\Runway\RunwayLights\NavigLight.p3d` is also a runway light ("REIL")
`Land_PortableHelipadLight_01_F`/`PortableHelipadLight_01_F.p3d`

`Land_runway_edgelight` there are 7 variants for this and they are all lowercase.

Reply:
The flush light is also lowercase. All lights I've seen are lowercase.

Reply:
```
{ 
    (boundingBox _x) params ["_a", "_b"];
    (_a distance2D _b) > 5
} count (player nearObjects ["Building", 50]);
```
Could also check size (2d, because a lot of lights are actually very long poles)
(0.06ms vs 0.038ms for 15 objects)

Reply:
That's a neat idea, though I would say a much lower number than 5. Bushes and such should affect roughness.

Reply:
"light" is faster and should probably work for most objects we want to filter
we can always look at size checking later